provider/docker.go

```diff
@@ -214,6 +215,9 @@ func (provider *Docker) Provide(configurationChan chan<- types.ConfigMessage, po
 					}
 					eventHandler.Handle("start", startStopHandle)
 					eventHandler.Handle("die", startStopHandle)
+					eventHandler.Handle("health_status: healthy", startStopHandle)
```

Comment:
The way `health_status` event are sent via the docker api is a little buggy, see https://github.com/docker/docker/pull/25800 👼